frontend/src/pages/project.jsx

```diff
@@ -0,0 +1,130 @@
+import { TransformRounded as Logo, LockOutlined as LockOutlinedIcon, Error as Icon } from '@material-ui/icons';
+import { Response, Project as ProjectFetch } from "../utils/connect"
+import { RouteComponentProps } from 'react-router-dom';
+import { Project, GetProjectResponse, ProjectDetails } from "../utils/connect"
+import { ModelsSearchComponent } from '../components/ModelsSearch'
+
+const styles = (theme) =>
+	createStyles({
+		cover: {
+			width: "100%",
+			padding: "40px 0",
+			display: "flex",
+			flexDirection: "column",
+			justifyContent: "space-between"
+		},
+		coverItem: {
+			marginBottom: theme.spacing.unit
+		},
+		button: {
+			margin: theme.spacing.unit,
+		},
+		row: {
+			display: "flex",
+			flexDirection: "row",
+			marginTop: "10px",
+			"& *:first-child":{
+				marginLeft:0
+			}
+		}
+	});
+
+class ProjectComponent extends React.Component {
+	lastReq = "";
+	constructor(props) {
+		super(props);
+		this.state = {
+			status: "loading",
+			id: 0,
+			name: "please wait",
+			description: "please wait",
+			repoUrl: "please wait",
+			allParameters: ["please wait"],
+			allHiperParameters: ["please wait"],
+			allModelTags: ["please wait"],
+			allModelNames: ["please wait"],
+			allMetrics: ["please wait"]
+		}
+	}
+	getProject = async () => {
+		const urlId = this.props.match.params.projectId;
+		this.setState({ status: "loading" })
+		const project = await ProjectFetch.getProject(Number(urlId));
+
+		if (!project.successful) {
+			this.setState({ status: "failed" });
+			return;
+		}
+
+		this.setState({
+			id: project.id,
+			name: project.name,
+			description: project.description,
+			repoUrl: project.repoUrl,
+			allParameters: project.allParameters,
+			allHiperParameters: project.allHiperParameters,
+			allModelTags: project.allModelTags,
+			allModelNames: project.allModelNames,
+			allMetrics: project.allMetrics,
+			status: "loaded"
+		})
+	}
+	componentDidUpdate(prevProps, prevState) {
+		const urlId = this.props.match.params.projectId;
+		const prevUrlId = prevProps.match.params.projectId;
+		if (urlId !== prevUrlId) this.getProject();
+	}
+	componentDidMount() {
+		this.getProject();
+	}
+	render() {
+		const {classes} = this.props;
+		const {state} = this;
+		if (this.state.status === "loaded") {
+			const project = this.state;
+			return (
+				<div>
+					<div className={classes.cover}>
+						<Typography className={classes.coverItem} variant={"h3"}>{state.name}</Typography>
+						<Typography className={classes.coverItem} variant={"h5"}>{state.description}</Typography>
+						<div className={classes.row}>
+							<Button variant="outlined" size="small" color="primary" className={classes.button}>
+								Code Repo
+							</Button>
+							<Button variant="outlined" size="small" color="primary" className={classes.button}>
+								Issue Tracker
+							</Button>
+							<Button variant="outlined" size="small" color="primary" className={classes.button}>
+								Live Demo
```

Comment:
What would the `Live Demo` button be? 
Is it possibly showcasing a list of currently deployed models? (something I plan on implementing in the near future)

Reply:
I expect some projects might have continuously deployed demos.